addons/pos_self_order/static/src/app/self_order_service.js

```diff
@@ -221,11 +221,23 @@ export class SelfOrder extends Reactive {
         };
 
         if (Object.entries(selectedValues).length > 0) {
-            values.attribute_value_ids = Object.values(selectedValues).map((a) => {
-                const attrVal = this.models["product.template.attribute.value"].get(a);
-                values.price_extra += attrVal.price_extra;
-                return ["link", attrVal];
-            });
+            values.attribute_value_ids = Object.entries(selectedValues).reduce(
+                (acc, [attributeId, options]) => {
```

Comment:
It seems `options` can be of two types. May I know what are they?

Reply:
yes one is `string` for example beacon burger with selection of slides 
and `object` for example pizza margarita with selection of extras 

ref video: https://www.awesomescreenshot.com/video/31857787?key=5f443d420da8f2c9fca20493f79fd8f1

Reply:
Hello @caburj 
Could you please check?
Thanks!